package.json

```diff
@@ -68,8 +68,7 @@
       "yarn lint:js"
     ],
     "packages/*/package.json": [
-      "yarn lint:pkg-json",
-      "yarn check-licenses"
+      "yarn lint:pkg-json"
```

Comment:
is including this json file intended?

Reply:
Yes. It seems this `check-licenses` is not working correctly. After talking to Kuba, he suggested removing it.